test/QuadForm/Quad/ZLattices.jl

```diff
@@ -347,6 +341,39 @@ end
   @test ambient_space(N) === ambient_space(L)
   @test rank(N) == 0
   @test basis_matrix(invariant_lattice(L, identity_matrix(QQ, 2))) == basis_matrix(L)
+  
+  randlist = rand(2:70,10)
```

Comment:
```suggestion
  randlist = rand(2:20,10)
```
There is no need to test so many or big lattices. If it works for `2:10` it works.